haystack/schema.py

```diff
@@ -1392,6 +1393,42 @@ def save(self, out_dir: Union[str, Path], **to_csv_kwargs):
             to_csv_kwargs = {**default_to_csv_kwargs, **to_csv_kwargs}
             df.to_csv(target_path, **to_csv_kwargs)
 
+    def save_excel(self, out_file: Union[str, Path], **to_excel_kwargs):
+        """
+        Saves the evaluation result in Excel format.
+        The result of each node is saved in a separate sheet of the out_file file.
+        multilabel_id column of created file is saved as string to prevent automatic rounding of large numbers
```

Comment:
```suggestion
       The multilabel_id column of the file is saved as a string to prevent Excel from automatic rounding of large numbers (more than 15 digits).
```